healthcheck/healthcheck.js

```diff
@@ -1,7 +1,20 @@
 const constants = require('../utils/constants')
 
-const getHealthStatus = () => {
-  return constants.CONSTANTS.HEALTHCHECK;
+const getHealthStatus = (req) => {
+  const port = getPort(req);
+
+  const response = {
+    Port: port,
+    Application: constants.CONSTANTS.HEALTHCHECK.Application,
+    Status: constants.CONSTANTS.HEALTHCHECK.Status
+  }
+  return response;
+}
+
+function getPort(req) {
+  let hostArr = req.headers.host.split('');
+  const port = hostArr.slice(-4);
```

Comment:
This code will get port value from Url. It will not work properly for DNS uri that may not include port value. For example **https://magic-marks-api.herokuapp.com/healthcheck**